packages/components/pie-card/src/index.ts

```diff
@@ -53,6 +53,9 @@ export class PieCard extends LitElement implements CardProps {
     @validPropertyValues(componentSelector, paddingValues, undefined)
     public padding?: CardProps['padding'];
 
+    @queryAssignedElements({ flatten: true })
+    private assignedElements!: HTMLElement[];
```

Comment:
In this case I would probably caution against using `!` because it's possible for there to be no elements passed in, so we want to be checking for and handling that case

Reply:
Fixed 👍